frontend/src/metabase/query_builder/components/expressions/Editor/Editor.tsx

```diff
@@ -4,48 +4,50 @@ import cx from "classnames";
 import { useCallback, useMemo, useRef, useState } from "react";
 import { useMount } from "react-use";
 import { t } from "ttag";
+import _ from "underscore";
```

Comment:
When I try to add a custom column, the editor is autofocused, but when I try to add a custom filter it's not.